packages/xml-views-validation/src/validators/attributes/unknown-xmlns-namespace.ts

```diff
@@ -0,0 +1,46 @@
+import { XMLAttribute } from "@xml-tools/ast";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import { findSymbol } from "@ui5-language-assistant/semantic-model";
+import { UnknownNamespaceInXmlnsAttributeValueIssue } from "../../../api";
```

Comment:
import order (can you try integrating the eslint rule? I did not succde)

Reply:
I want to try but I don't think I'll have time this sprint

Reply:
Maybe I can find time to try again during some useless meeting 😄